src/main/java/org/cascadebot/cascadebot/commands/music/KaraokeCommand.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2019 CascadeBot. All rights reserved.
```

Comment:
*Cough* I think we're living in 2020 now.. 
I know other files don't have this, but you should